src/main/java/us/jonathans/Config.java

```diff
@@ -2,4 +2,5 @@
 
 public class Config {
     public static final String APP_NAME = "Jacala (Mancala in Java)";
+    public static final String leaderboardFrame= "Leaderboard";
```

Comment:
Use uppercase for constants and give it a more descriptive name like `LEADERBOARD_FRAME_NAME`